ts-web-component/src/component/todo-item.ts

```diff
@@ -54,31 +57,44 @@ export default class TodoItem extends HTMLElement {
     return this._todoID;
   }
 
-  disconnectedCallback() {}
+  get text() {
+    return this.getAttribute("text") ?? "";
```

Comment:
One of the nice things about custom elements/web components is that you can let the DOM hold the source of truth.